events/EventQueue.cpp

```diff
@@ -19,35 +19,42 @@
 #include "mbed.h"
 
 
-EventQueue::EventQueue(unsigned event_size, unsigned char *event_pointer) {
+EventQueue::EventQueue(unsigned event_size, unsigned char *event_pointer)
+{
     if (!event_pointer) {
```

Comment:
My complaint is that squiggly brackets are still inconsistent.

We need to choose one of these:
``` cpp
class Thing {
    int dothing() {
        if (thing) {
            blahblahblah;
        }
    }
};
```

``` cpp
class Thing
{
    int dothing()
    {
        if (thing)
        {
            blahblahblah;
        }
    }
};
```

Not both, what's the point of this?

``` cpp
class Thing {
    int dothing()
    {
        if (thing) {
            blahblahblah;
        }
    }
};
```

Reply:
This is how they have been, attached but not for functions, see our coding style since mbed 2.

Reply:
My point is that even though it's in our style guide, we're not using it consistently (see above numbers). Perhaps because it's confusing.

Reply:
You are the first to mention this that I am aware of. 

Time to set it correctly now in all files and have it checked.